test/mocks/upstream/cluster_manager.cc

```diff
@@ -20,7 +20,8 @@ MockClusterManager::MockClusterManager()
       cluster_load_report_stat_names_(*symbol_table_),
       cluster_circuit_breakers_stat_names_(*symbol_table_),
       cluster_request_response_size_stat_names_(*symbol_table_),
-      cluster_timeout_budget_stat_names_(*symbol_table_) {
+      cluster_timeout_budget_stat_names_(*symbol_table_),
+      common_lb_config_pool_(SharedPool::ObjectSharedPool<const envoy::config::cluster::v3::Cluster::CommonLbConfig, MessageUtil, MessageUtil>(dispatcher_)){
```

Comment:
I don't think you need to copy construct this.

Reply:
Done